spec/mailboxes/ticket_mailbox_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe TicketMailbox, type: :mailbox do
+  let(:body) { Faker::Lorem.paragraphs(number: 2).join(',') }
+  let(:subject) { Faker::Internet.name }
```

Comment:
This let name may be confusing since we use `subject` for the `test subject` (this let could be renamed to `mail_subject` or something).

It looks like `process(mail)` could be the test subject here, so you could define `subject { process(mail) }`